### PR TITLE
Sanitize IP zookeeper/enable_and_wait.py

### PR DESCRIPTION
**Pull request by whokilleddb**
**Description**
This PR fixes a minor security issue that can lead to os code injection.

**Verification** 
The function was tested locally with various valid and invalid inputs and it produced the appropriate results.

> PS: Can the maintainers please reserve a CVE for this is they deem the issue fit?

### CODE REVIEW

File: tools/zookeeper/enable_and_wait.py [hunk 1]
```diff
@@ -3,10 +3,15 @@
...
 import time
+from socket import gethostbyname
 from tools.zookeeper import Zookeeper
 
 def is_online(ip):
-  return (os.system(f"ping -c 1 {ip} > /dev/null") == 0)
+  try:
+      addr = gethostbyname(ip)
+      return (os.system(f"ping -c 1 {addr} > /dev/null") == 0)
+  except:
```
Comment (robbederks) [inline tools/zookeeper/enable_and_wait.py:13]: Mind fixing the bare except? this is causing the static analysis to fail.

Comment (whokilleddb) [reply tools/zookeeper/enable_and_wait.py:13]: Thanks for the feedback @robbederks 

I'll do it right away.

**Comment by whokilleddb** [comment]
Hey @robbederks fixed the bug and even passed the test cases 😀 

Also, can you please look into #24015? I am a part of the said community of security researchers and the `SECURITY.md` would be really helpful 😄

**Comment by adeebshihadeh** [comment]
> Hey @robbederks fixed the bug and even passed the test cases grinning
> 
> Also, can you please look into #24015? I am a part of the said community of security researchers and the `SECURITY.md` would be really helpful smile

Feel free to email me or robbe for now (first_name@comma.ai).

**Comment by JamieSlome** [comment]
With regards to #24015, it was to share the contents of this vulnerability report, i.e. alluding to code injection in `enable_and_wait.py`. 

The report can be found here (submitted by @whokilleddb):
https://www.huntr.dev/bounties/8c089464-637c-49a6-9839-0929740f605c

It is currently private and is only visible to maintainers with repository write permissions. Let me know if you would like me to make it publicly visible 👍

**Comment by JamieSlome** [comment]
Furthermore, we can arrange a CVE for you (@robbederks @adeebshihadeh) if required, as the vulnerability was initially reported via huntr.dev 👍

**Comment by gregjhogan** [comment]
It doesn't make sense to open a CVE for this issue, and you probably damage your credibility as a security researcher with your approach.  It might be a good idea to put some effort into better understanding the software you are looking at before popping the CVE question.  Join our discord server if you care to learn more.

File: tools/zookeeper/enable_and_wait.py [hunk 2]
```diff
@@ -3,10 +3,15 @@
 import os
 import sys
 import time
+from socket import gethostbyname, gaierror
 from tools.zookeeper import Zookeeper
 
 def is_online(ip):
-  return (os.system(f"ping -c 1 {ip} > /dev/null") == 0)
+    try:
```
Comment (geohot) [inline tools/zookeeper/enable_and_wait.py:10]: Why didn't the linter catch this wrong indentation?

**Pull request merged**